docs/reference/setup/upgrade.asciidoc

```diff
@@ -7,6 +7,7 @@ Before upgrading Elasticsearch:
 
 * Consult the <<breaking-changes,breaking changes>> docs.
 * Test upgrades in a dev environment before upgrading your production cluster.
+* Remove your current plugins, then install the latest after upgrading.
```

Comment:
You can't upgrade while ES is up, so it would be:

- shut down ES
- upgrade ES
- upgrade plugins
- (other upgrade things)
- restart ES